<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QDialog" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainScreen</string>
  </property>
  <widget class="QGroupBox" name="Input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Ipput</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="method">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Recognition Method</string>
    </property>
    <widget class="QRadioButton" name="pca">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>PCA</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="lda">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>LDA</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enhanced Local Texture </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="use_ELT">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Use</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>441</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Data train</string>
    </property>
    <widget class="QRadioButton" name="profile_face">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile face</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="fronta_face">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Frontal face</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Test</string>
    </property>
    <widget class="QLineEdit" name="path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="brow">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>98</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="OK">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ReSult">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Result</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
